aries-site/src/examples/components/menu/MenuToolBarExample.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Menu } from 'grommet';
+import {
+  FilterControls,
+  FiltersProvider,
+} from '../../templates/FilterControls';
+
+export const MenuToolBarExample = () => (
+  <FiltersProvider>
```

Comment:
We already have an equivalent example on https://design-system.hpe.design/components/button?q=button#toolbar-button -- can we just point to that exmaple on the MDX page?

Reply:
good point one less file to have